### DISCUSSION TITLE
Multiple screens ?

### DISCUSSION THREAD

**Discussion by BuissonNeth** [vzhd1701/gridplayer#89] [Q&A]
Hello ! 
Do you have a feature to play synchronously multiple video files on multiple screens ?
If not, do you plan on making such a feature ? 

Thanks for your app !

**Comment by vzhd1701** [comment]
This feature is not supported yet. Yes, it is planned for the future.

**Comment by BuissonNeth** [comment]
Thanks for the quick answer !

**Comment by Tany9696** [comment]
hi thanks for briliant app , i have a question , how can i open 2 grid player at the same time?
